server/Gopkg.toml

```diff
@@ -5,4 +5,8 @@
 
 [[constraint]]
   name = "github.com/mattermost/mattermost-server"
-  version = "~5.3.0"
+  branch = "master"
```

Comment:
Once 5.4.0-rc1 is cut, we can change this.

Reply:
Could we just point this to the `release-5.4` branch instead?

Reply:
Sure